src/Sentry/SentryExceptionExtensions.cs

```diff
@@ -18,6 +20,36 @@ public static class SentryExceptionExtensions
     public static void AddSentryTag(this Exception ex, string name, string value)
         => ex.Data.Add($"{MainExceptionProcessor.ExceptionDataTagKey}{name}", value);
 
+    /// <summary>
+    /// Recursively enumerates all <see cref="AggregateException.InnerExceptions"/> and <see cref="Exception.InnerException"/>
+    /// Not for public use.
+    /// </summary>
+    public static IEnumerable<Exception> EnumerateChainedExceptions(this Exception exception, SentryOptions options)
```

Comment:
This not being for public use but part of the `global` namespace don't go well together.
Should we remove the note on the xml doc?

Or perhaps this is best not being an extension method of Exception? If it's under a different namespace it will already help not show up on the user's intellisense. The goal is to share the built-in behavior with the Unity IL2CPP support

Reply:
I'm ok with it as is, since it includes a `SentryOptions` parameter - we're not likely to step on anyone.  We can hide it from intellisense with `EditorBrowsable`